connector/src/main/java/com/google/cloud/spark/bigquery/v2/BigQueryDataSourceReader.java

```diff
@@ -162,6 +163,11 @@ public boolean enableBatchRead() {
     ReadSessionResponse readSessionResponse =
         readSessionCreator.create(tableId, selectedFields, filter);
     ReadSession readSession = readSessionResponse.getReadSession();
+    logger.info(
+        "Created read session for {}: {} for application id: {}",
+        tableId.toString(),
+        readSession.getName(),
+        SparkContext.getOrCreate().applicationId());
```

Comment:
`SparkBigQueryConnectorModule` has a reference to the SparkSession. Please use it to inject the application ID to the DataSourceReader upon creation, instead of calling `SparkContext.getOrCreate().applicationId()`

Reply:
done.